pt/lessons/specifics/ecto.md

```diff
@@ -0,0 +1,368 @@
+---
+layout: page
+title: Ecto
+category: specifics
+order: 2
+lang: pt
+---
+
+Ecto é um projecto oficial do Elixir que fornece um invólucro de base de dados e
+linguagem integrada de consulta. Com Ecto podemos criar *migrations*, definir
+modelos, inserir e actualizar registos, e consultá-los.
+
+## Tabela de Conteúdos
+
+- [Instalação](#setup)
```

Comment:
The anchor links shuld be the same as the name.

ex:

[Instalação](#instalacao)